torch/csrc/jit/cuda/cuda.h

```diff
@@ -22,7 +21,6 @@ class CUDAStream final : public CustomClassHolder {
         c10::cuda::getStreamFromPool(static_cast<int>(priority), device_index));
   }
 
-  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
   CUDAStream(c10::cuda::CUDAStream s) {
```

Comment:
Hmm, CUDAStream is trivially copyable?

Reply:
This lint is about member initialisation